### PR TITLE
localize reliability.md and serverless.md

### PR DESCRIPTION
**Pull request by Submarinee**
localize reliability and serverless.md

### CODE REVIEW

**Review by Rocksnake — APPROVED** [review]

File: content/zh/serverless.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+Title: Serverless
+Status: Completed
+Category: Technology
+---
+
+## 是什么
+Serverless 是一种云原生开发模型，允许开发人员构建和运行应用程序，而无需管理服务器。 Serverless中仍有服务器，但它们被 [抽象](/abstraction/) 出来，远离应用程序开发。 云提供商处理配置、维护和 [扩展](/scalability/) 服务器基础架构的日常工作。 开发人员可以简单地将他们的代码打包在 [容器](/container/) 中进行部署。 部署后，Serverless应用程序会响应需求并根据需要自动扩展和缩减。 公共云提供商的Serverless产品通常通过事件驱动的执行模型按需计量。 因此，当无服务器功能处于空闲状态时，它不会花费任何费用。
```
Comment (Rocksnake) [inline content/zh/serverless.md:8]:
```suggestion
Serverless 是一种云原生开发模型，允许开发人员构建和运行应用程序，而无需管理服务器。 Serverless 中仍有服务器，但它们被 [抽象](/abstraction/) 出来，远离应用程序开发。 云提供商处理配置、维护和 [扩展](/scalability/) 服务器基础架构的日常工作。 开发人员可以简单地将他们的代码打包在 [容器](/container/) 中进行部署。 部署后，Serverless 应用程序会响应需求并根据需要自动扩展和缩减。 公共云提供商的 Serverless 产品通常通过事件驱动的执行模型按需计量。 因此，当无服务器功能处于空闲状态时，它不会花费任何费用。
```

File: content/zh/serverless.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## 解决的问题
+在标准的 [基础设施即服务 (IaaS)](/infrastructure_as_a_service/) [云计算](/cloud_computing/) 模型下，用户预先购买容量单位，这意味着您需要向公共云提供商支付永远在线的服务器组件的费用来运行您的应用程序。
+用户有责任在高需求时扩展服务器容量，并在不再需要该容量时缩减容量。 即使在不使用应用程序时，运行应用程序所需的云基础设施也处于活动状态。
```
Comment (Rocksnake) [inline content/zh/serverless.md:12]:
```suggestion
用户有责任在高需求时扩展服务器容量，并在不在需要该容量时缩减容量。 即使在不使用应用程序时，运行应用程序所需的云基础设施也处于活动状态。
```

File: content/zh/serverless.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## 如何帮助
+相比之下，使用Serverless架构，应用程序仅在需要时启动。 当事件触发应用程序代码运行时，公共云提供商会为该代码动态分配资源。 当代码完成执行时，用户停止支付。 除了成本和效率优势之外，Serverless还使开发人员从与应用程序扩展和服务器配置相关的日常和琐碎任务中解放出来。 借助Serverless，管理操作系统和文件系统、安全补丁、负载平衡、容量管理、扩展、日志记录和监控等日常任务都被交给云服务提供商。
```
Comment (Rocksnake) [inline content/zh/serverless.md:15]:
```suggestion
相比之下，使用 Serverless 架构，应用程序仅在需要时启动。 当事件触发应用程序代码运行时，公共云提供商会为该代码动态分配资源。 当代码完成执行时，用户停止支付。 除了成本和效率优势之外，Serverless 还使开发人员从与应用程序扩展和服务器配置相关的日常和琐碎任务中解放出来。 借助 Serverless，管理操作系统和文件系统、安全补丁、负载平衡、容量管理、扩展、日志记录和监控等日常任务都被交给云服务提供商。
```

Comment (Rocksnake) [inline content/zh/serverless.md:15]:
```suggestion
相比之下，使用Serverless架构，应用程序仅在需要时启动。 当事件触发应用程序代码运行时，公共云提供商会为该代码动态分配资源。 当代码执行完成后，用户就停止为资源付款。除了成本和效率优势之外，Serverless还使开发人员从与应用程序扩展和服务器配置相关的日常和琐碎任务中解放出来。 借助Serverless，管理操作系统和文件系统、安全补丁、负载平衡、容量管理、扩展、日志记录和监控等日常任务都被交给云服务提供商。
```

Comment (Rocksnake) [reply content/zh/serverless.md:15]: I have a suggestion, would it be better to translate it this way. This may also need other approvers to confirm.

**Review by hanyuancheung — APPROVED** [review]
lgtm

**Pull request merged**